documentation/modules/snip_qemu-guest-agent.adoc

```diff
@@ -0,0 +1,7 @@
+:_content-type: SNIPPET
+
+VMware only: In cold migrations, in situations in which a package manager cannot be used during the migration, {project-short} does not install the `qemu-guest-agent` daemon on the migrated VMs. This has some impact on the functionality of the migrated VMs, but overall, they are still expected to function.
+
+To enable {project-short} to automatically install `qemu-guest-agent` on the migrated VMs, ensure that your package manager can install the daemon.
```

Comment:
```suggestion
To enable {project-short} to automatically install `qemu-guest-agent` on the migrated VMs, ensure that your package manager can install the daemon during the first boot of the VM after migration.
```

IMHO we can be more transparent here and explain when exactly during the migration process we need the package manager to be available.

Reply:
Done.